### ISSUE TITLE
Data augment and dataloader

### ISSUE THREAD

**Issue by ballinhuang** [aliprf/KD-Loss#3]
The Training Student Networks has opened the use_augmneted
```
StudentTrainer(dataset_name=DatasetName.w300, use_augmneted=True)
```
But the Preparing Data section didn't mention how to generate the augmented data.

And can't find the module TFRecordUtility used in the training process.
https://github.com/aliprf/KD-Loss/blob/master/student_train.py#L201
